main.cpp

```diff
@@ -270,6 +272,7 @@ int main (int argc, char **argv)
 	//init stuff
 	mode = MODE_NORMAL;
 	in_macro = 0;
+	quiet_output = true;

```

Comment:
Wouldn't this cause `quiet_output` to be true even if the `-q` flag is unset?

Reply:
Oops...